pkg/framework/aries/factory/protocol/protocol_factory.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package protocol
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/exchange"
```

Comment:
ditto: https://github.com/hyperledger/aries-framework-go/pull/88/files#r313588529

Reply:
[changed the abstraction](https://github.com/hyperledger/aries-framework-go/pull/88#discussion_r313604198)